pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/pancan/PancanPrequalAddChildV2.java

```diff
@@ -19,9 +19,11 @@
 
 import java.nio.file.Path;
 import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
 
 @Slf4j
-public class PancanPrequalV2 implements CustomTask {
+public class PancanPrequalAddChildV2 implements CustomTask {
```

Comment:
Renamed existing SB task (same ticket, same release) to include updated scope changes